learn/tanssi/network-features/index.md

```diff
@@ -1,8 +1,15 @@
 ---
 title: Network Features
 description: Learn about Tanssi general network features such as Staking, Governance and Treasury, and how each help Tanssi becoming a fully decentralized Appchain orchestrator for the Web3 ecosystem.
-template: subsection-index-page.html
-hide: 
- - toc
- - feedback
+icon: material-blur
+template: index-page.html
 ---
+
+# Network Features
+
+The network features in Tanssi are designed to provide the foundational mechanisms that support decentralization, security, and scalability. From securing appchains through staking to future capabilities, this section outlines the core elements that empower Tanssi’s appchain infrastructure.
+
```

Comment:
This is a super teeny tiny nit pick. But it's good to be aware of the spacing in the markdown files and only have one line break between elements.

```suggestion
```